app/components/sections/hero-section.tsx

```diff
@@ -93,6 +93,7 @@ function HeroSection({
           {imageProps ? (
             <motion.img
               {...imageProps}
+              suppressHydrationWarning={true}
```

Comment:
Why is this necessary?

Reply:
i get this on localhost
```
react-dom.development.js:86 Warning: Prop `alt` did not match. Server: "Illustration of Kody the Koala skiing surrounded by green leaves, a battery, two skies, a one-wheel, a solar panel, and a recycle logo." Client: "Illustration of Kody the Koala standing on a snowboard surrounded by green leaves, a battery, two skies, a one-wheel, a solar panel, and a recycle logo."
    at img
    at MotionComponent (http://localhost:3000/build/_shared/chunk-BKXH5CFN.js:285:40)
    at div
    at div
    at header
    at Grid2 (http://localhost:3000/build/_shared/chunk-ZH4MEJF3.js:40:3)
    at HeroSection (http://localhost:3000/build/_shared/chunk-BKXH5CFN.js:7553:3)
    at div
    at IndexRoute (http://localhost:3000/build/routes/index-EVZDMZW4.js:683:16)
    at RemixRoute (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:5762:3)
    at RenderedRoute (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3483:5)
    at RenderErrorBoundary (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:4041:9)
    at Outlet (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3825:26)
    at body
    at html
    at App (http://localhost:3000/build/_shared/chunk-FSYMWBNO.js:7734:16)
    at ThemeProvider (http://localhost:3000/build/_shared/chunk-PS2XUJXE.js:54:3)
    at TeamProvider (http://localhost:3000/build/_shared/chunk-FSYMWBNO.js:6138:3)
    at AppWithProviders (http://localhost:3000/build/_shared/chunk-FSYMWBNO.js:7855:16)
    at RemixRoute (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:5762:3)
    at RenderedRoute (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3483:5)
    at RenderErrorBoundary (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:4041:9)
    at DataRoutes (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3753:5)
    at Router (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3832:15)
    at RouterProvider (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:3702:5)
    at RemixErrorBoundary (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:5352:5)
    at RemixBrowser (http://localhost:3000/build/_shared/chunk-HIPMZLZ5.js:7277:40)
```

Reply:
That warning is an indication of a real problem. That alt text should be exactly the same in the client and server. If it's not, the user will see a flash of the wrong image. I can look at this later if you don't get to it first, but we need to determine which image/alt text to use in the loader so it remains consistent.

Reply:
I don't see it on the real site, only on my local dev build. 
Maybe it is only window issue. Maybe it rekated to \\ instead of / somewher?

Reply:
Let's remove it for now and I'll look into it later.

Reply:
https://github.com/kentcdodds/kentcdodds.com/blob/94ae5849251a3a48e38d8253854730b712cf615b/app/routes/index.tsx#L87-L89

`getRandomFlyingKody(data.kodyTeam)` return random image.

that is what's causing this issue. can i move the line to the loader ?

Reply:
Ah, yeah, let's move that to the loader. Not sure how that got there! Thank you for finding it :)